doc/classes/TileSetScenesCollectionSource.xml

```diff
@@ -6,6 +6,17 @@
 	<description>
 		When placed on a [TileMap], tiles from [TileSetScenesCollectionSource] will automatically instantiate an associated scene at the cell's position in the TileMap.
 		Scenes are instantiated as children of the [TileMap] when it enters the tree. If you add/remove a scene tile in the [TileMap] that is already inside the tree, the [TileMap] will automatically instantiate/free the scene accordingly.
+		[b]Note:[/b] Scene tiles all occupy one tile slot and instead use alternate tile ID to identify scene index. [method TileSetSource.get_tiles_count] will always return [code]1[/code]. Use [method get_scene_tiles_count] to get a number of scenes in a [TileSetScenesCollectionSource].
+		Use this code if you want to find the scene path at a given tile in [TileMapLayer]:
+		[codeblock]
+		var source_id = tile_map_layer.get_cell_source_id(Vector2(x, y))
+		if source_id &gt; -1:
+		    var scene_source = tile_map_layer.tile_set.get_source(source_id) as TileSetScenesCollectionSource
+		    if scene_source:
```

Comment:
Line would be too long otherwise. Also makes it simpler to read.
```suggestion
		    var scene_source = tile_map_layer.tile_set.get_source(source_id)
		    if scene_source is TileSetScenesCollectionSource:
```